### ISSUE TITLE
GitHub wants me to update a token

### ISSUE THREAD

**Issue by Pirol62** [hacs/integration#2070]
## System Health

version | core-2021.5.4
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
virtualenv | false
python_version | 3.8.9
os_name | Linux
os_version | 4.19.93-v7
arch | armv7l
timezone | Europe/Berlin

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
Github API Calls Remaining | 3079
Installed Version | 1.12.3
Stage | running
Available Repositories | 826
Installed Repositories | 31

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | true
-- | --
subscription_expiration | 29. Mai 2021, 2:00
relayer_connected | true
remote_enabled | false
remote_connected | true
alexa_enabled | true
google_enabled | false
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Home Assistant Supervisor</summary>

host_os | HassOS 3.11
-- | --
update_channel | stable
supervisor_version | supervisor-2021.04.3
docker_version | 19.03.5
disk_total | 28.6 GB
disk_used | 6.4 GB
healthy | true
supported | true
board | rpi3
supervisor_api | ok
version_api | ok
installed_addons | Samba share (9.3.1), Duck DNS (1.12.5), File editor (5.3.1), SQLite Web (3.1.0), Dropbox Sync (1.3.0), Check Home Assistant configuration (3.7.1), deCONZ (6.8.0), SSH & Web Terminal (8.1.1), Mosquitto broker (5.1.1), HiSense Air Conditioners (0.3.9)

</details>

<details><summary>Lovelace</summary>

dashboards | 5
-- | --
resources | 19
views | 29
mode | storage

</details>

### Checklist

- [X] I'm running the newest version of HACS <https://github.com/hacs/integration/releases/latest>
- [X] I have enabled debug logging for my installation.
- [X] I have filled out the issue template to the best of my ability.
- [X] I have read <https://hacs.xyz/docs/issues>
- [X] This issue is related to the backend (integration part) of HACS.
- [X] This issue only contains 1 issue (if you have multiple issues, open one issue for each issue).

### Describe the issue

GitHub want's me to update the access token.
I've never created one as I remember.
Does it have any influence on the integration?

![image](https://user-images.githubusercontent.com/48820003/118943474-f2b05a80-b953-11eb-8bd7-e17166923847.png)

### Reproduction steps

1. received eMail from GitHub
2.
3.
...

### Debug logs

```text
not acquired
```

**Comment by ludeeus** [comment]
See https://github.com/hacs/integration/issues/2014

**Comment by Pirol62** [comment]
@ludeeus Hi, there are two things different here:
1. I cannot deinstall HACS because it is not existent in integrations view but in the integrations group in HACS
![image](https://user-images.githubusercontent.com/48820003/118983529-d543b680-b97c-11eb-8d7d-d725c6e796f2.png)
So I can only reinstall HACS what I already did.
2. My link works. When I click on it and enter passwaord, I got this UI: 
![image](https://user-images.githubusercontent.com/48820003/118983846-2653aa80-b97d-11eb-814f-3dbb422f0b05.png)

So if I would know the scope, I could simply update the token.
What way is the best now?
